xbmc/windowing/gbm/DRMAtomic.cpp

```diff
@@ -140,7 +140,7 @@ void CDRMAtomic::DrmAtomicCommit(int fb_id, int flags, bool rendered, bool video
       return;
     }
 
-    if (!AddCrtcProperty(m_req, m_crtc->crtc->crtc_id, "ACTIVE", 1))
+    if (!AddCrtcProperty(m_req, m_crtc->crtc->crtc_id, "ACTIVE", m_active ? 1 : 0)
```

Comment:
`? 1 : 0))`

Reply:
Sorry, was just a one-liner, I forgot to locally build it first. But I see you already fixed it in master, thanks!

Reply:
Not your fault, we have CI for that, but should be also wait for it to finish.